react-frontend/src/shared-components/peerly-logo/LogoComponent.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+import { Row } from "core-components/grid/GridComponent";
+import PlusSign from "shared-components/plus-sign/PlusSign";
+
+const Plus = styled.div`
```

Comment:
[Every component should follow “single responsibility principle”](https://github.com/joshsoftware/peerly/wiki/React-Development-Guidelines)